jest.config.js

```diff
@@ -1,8 +1,12 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const { createConfig } = require('@openedx/frontend-build');
+const path = require('path');
 
 module.exports = createConfig('jest', {
   setupFiles: [
     '<rootDir>/src/setupTest.js',
   ],
+  moduleNameMapper: {
```

Comment:
In `src/data/validation/email.js` as an isolated example locally, on `master` after installing `lodash-es`, I replaced the original `lodash` usage with `lodash-es` to observe the test failure around the `SyntaxError` with `export`. Then, modifying the `jest.config.js` made the test pass (without any other changes, like the `__mocks__` file):

```js
const config = createConfig('jest', {
  setupFiles: [
    '<rootDir>/src/setupTest.js',
  ],
});

config.transformIgnorePatterns = ['/node_modules/(?!(lodash-es|@(open)?edx)/)'];

module.exports = config;
```